d2l-floating-buttons.js

```diff
@@ -0,0 +1,225 @@
+/**
+`d2l-floating-buttons`
+Polymer-based web component for floating workflow buttons
+
+@demo demo/floating-buttons.html d2l-floating-buttons
+*/
+import '../@polymer/polymer/polymer-legacy.js';
+
+import '../d2l-colors/d2l-colors.js';
+import '../@polymer/iron-media-query/iron-media-query.js';
+import './d2l-button-shared-styles.js';
+import { Polymer } from '../@polymer/polymer/lib/legacy/polymer-fn.js';
+import { html } from '../@polymer/polymer/lib/utils/html-tag.js';
```

Comment:
Interesting that it pulled in this tagged template function.   Adding `strip-whitespace` to the original would result in this component being handled similar to the rest.  Originally I was trying to be consistent with this (and the `strip-whitespace` is typically preferred anyway), but I'm not sure it makes any difference.

Reply:
Oh I didn't notice that this component was missing `strip-whitespace`. I can add it and rerun the modulizer